src/Credentials.js

```diff
@@ -233,4 +245,25 @@ const configNetworks = (nets) => {
   return nets
 }
 
+const padMessage = (message) => {
+  const INTERVAL_LENGTH = 50
```

Comment:
why 50?

Reply:
This is kind of arbitrary. A too short interval would reveal too much about the message and a too long would result in too much overhead. 50 is simply what me and @christianlundkvist arrived at when talking about this.